### ISSUE TITLE
Memory leak! When disposing graph which shares selectionChannel

### ISSUE THREAD

**Issue by LucaCras** [entronad/graphic#201]
We use multiple graphs/charts, which share the same selectionChannel. When one of the charts is disposed (i.e. deleted), and performing a selection in the other chart, the setState() method is called for the other (deleted) chart still. 

This probably points toward a memory leak.

Exact message:
```
[ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: 
setState() called after dispose(): _ChartState<RatePoint>#3e1b7(lifecycle state: defunct, not mounted)
This error happens if you call setState() on a State object for a widget that no longer appears in the widget tree 
(e.g., whose parent widget no longer includes the widget in its build). 
This error can occur when code calls setState() from a timer or an animation callback.
The preferred solution is to cancel the timer or stop listening to the animation in the dispose() callback. 
Another solution is to check the "mounted" property of this object before calling setState() to ensure the object is still in the tree.
This error might indicate a memory leak if setState() is being called because another object is retaining a reference to this State object after it has been removed from the tree. To avoid memory leaks, consider breaking the reference to this object during dispose().
#0      State.setState.<anonymous closure> (package:flutter/src/widgets/framework.dart:1097:9)
#1      State.setState (package:flutter/src/widgets/framework.dart:1132:6)
#2      _ChartState.repaint (package:graphic/src/chart/chart.dart:219:5)
#3      View.evaluate (package:graphic/src/chart/view.dart:93:14)
<asynchronous suspension>
#4      Dataflow.run.<anonymous closure> (package:graphic/src/dataflow/dataflow.dart:148:7)
<asynchronous suspension>
```

**Comment by BentEngbers** [comment]
New Error: from version `114`:
![CleanShot 2023-03-09 at 10 57 50](https://user-images.githubusercontent.com/16548374/223987709-9c03f6ac-3655-4133-935e-41ace276658a.png)

**Comment by entronad** [comment]
The dispose functions have been fixed in v1.1.0
